Header header
int32 level
int32 min_x
int32 max_x
int32 min_y
int32 max_y
